modules/cluster-logging-updating-logging-to-5-0.adoc

```diff
@@ -167,14 +167,14 @@ green  open   audit-000001
 +
 [source,terminal]
 ----
-$ oc get ds fluentd -o json | grep fluentd-init
+$ oc get ds collector -o json | grep collector-init
 ----
 +
-Verify that the output includes a `fluentd-init` container:
+Verify that the output includes a `collector-init` container:
 +
 [source,terminal]
 ----
-"containerName": "fluentd-init"
+"containerName": "collector-init"
```

Comment:
I am not sure if renamed to collector anything beyond 5.4. So fluentd should still apply on 5.0 (cc @jcantrill )